docs/developer/app-store/apps/smtp/overview.mdx

```diff
@@ -129,6 +129,7 @@ To run the application locally, follow the [documentation](../../../extending/ap
 - Emails use the default language of the Saleor instance. Using translations is not supported yet. [Issue link](https://github.com/saleor/apps/issues/726)
 - The order refunded event is not supported yet. [Issue link](https://github.com/saleor/apps/issues/728)
 - The fulfillment updated event is not supported yet. [Issue link](https://github.com/saleor/apps/issues/729)
+- Your SMTP server must respond in maximum 4 seconds. Othwerwise, the application return the timeout and your email will not be sent.
```

Comment:
```suggestion
- Your SMTP server must respond in a maximum 4 seconds. Otherwise, the application returns the timeout and your email will not be sent.
```

Please use Grammarly or a similar tool when writing updates to docs.